src/stylesheets/main.scss

```diff
@@ -427,3 +429,21 @@ pre .language-plaintext {
     font-weight: normal;
   }
 }
+
+// Extra DSD24 styles
+
+.app-prose-scope .dsd24-lead-paragraph {
+  @include govuk-font-size($size: 24);
+}
+
+.dsd24-headline-banner-container img {
+  width: 100%;
+  max-width: none;
+  margin: 0;
```

Comment:
Also to reduce CSS here, could we achieve the same result with the `.app-image--no-border` class on the image?

Reply:
Have used the override classes now :)